packages/apps/docs/src/components/CookieConsent/CookieConsent.tsx

```diff
@@ -0,0 +1,66 @@
+import { Notification, Text } from '@kadena/react-ui';
+
+import { notificationClass } from './styles.css';
+
+import { updateConsent } from '@/utils/analytics';
+import type { FC } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+
+export const CookieConsent: FC = () => {
+  const [cookieConsent, setCookieConsent] = useState<boolean | null>(null);
+
+  useEffect(() => {
+    const stickyValue = localStorage.getItem('cookie_consent');
+    if (stickyValue === null) return;
+    setCookieConsent(JSON.parse(stickyValue));
+  }, []);
+
+  useEffect(() => {
+    if (cookieConsent === null) return;
+    updateConsent(cookieConsent);
+  }, [cookieConsent]);
+
+  const handleAccept = useCallback(() => {
+    setCookieConsent(true);
+  }, []);
+
+  const handleReject = useCallback(() => {
+    setCookieConsent(false);
+  }, []);
+
+  if (cookieConsent !== null) return null;
+
+  return (
+    <div className={notificationClass}>
+      <Notification.Root
```

Comment:
can u add the wrapper (i think we use in Header)
the consent content is way to wide atm.
so the notification background is good. but the content needs to be as wide as the header content